src/api/lib/tasks/statistics/current.rake

```diff
@@ -6,6 +6,18 @@ namespace :statistics do
     information += ("Number of users: " + User.count.to_s + "\n")
     information += ("Number of projects: " + Project.count.to_s + "\n")
     information += ("Number of packages: " + Package.count.to_s + "\n")
+    
+    elements = HistoryElement::Request
+              .where(type: "HistoryElement::RequestAccepted")
+              .joins("INNER JOIN bs_requests ON bs_requests.id = history_elements.op_object_id")
+              .pluck('bs_requests.created_at', 'history_elements.created_at')
+    time = 0 # time in seconds
+    elements.each do |e|
```

Comment:
You could shorten this with [sum](https://ruby-doc.org/core-2.4.1/Enumerable.html#method-i-sum)